src/promptflow/promptflow/_sdk/_orm/run_info.py

```diff
@@ -158,3 +159,12 @@ def list(max_results: Optional[int], list_view_type: ListViewType) -> List["RunI
                 return [run_info for run_info in basic_statement.limit(max_results)]
             else:
                 return [run_info for run_info in basic_statement.all()]
+
+    @staticmethod
+    @sqlite_retry
+    def delete(name: str) -> None:
+        with mgmt_db_session() as session:
+            run_info = session.query(RunInfo).filter(RunInfo.name == name).first()
```

Comment:
as this is in ORM level operation, I prefer raise an error if `run_info` is `None`, which means the delete operation is failed/invalid.